release/pipelines/windows-efi-installer/README.md

```diff
@@ -110,6 +110,47 @@ EOF
 oc create -f - <<EOF
 apiVersion: tekton.dev/v1
 kind: PipelineRun
+metadata:
+    generateName: windows10-installer-run-
+spec:
+    params:
+    -   name: winImageDownloadURL
```

Comment:
e.g. here between the `-` and `name`, also it is using 4 spaces for indendation.

Reply:
This is done by ansible

Reply:
Weird but OK...